gnmvidispine/vs_acl.py

```diff
@@ -1,4 +1,4 @@
-from vidispine_api import VSApi
+from .vidispine_api import VSApi
```

Comment:
does this work in Py2 and py3? are there tests on this file?

Reply:
It should do. The same syntax works with files tested in the unit tests under Python 2. 

There are no direct tests for this file. However, it may be imported during other tests.